capsules/src/adc.rs

```diff
@@ -653,16 +653,18 @@ impl<'a> AdcVirtualized<'a> {
         processid: ProcessId,
     ) -> Result<(), ErrorCode> {
         if channel < self.drivers.len() {
-            self.apps
-                .enter(processid, |app, _| {
-                    if self.current_app.is_none() {
-                        self.current_app.set(processid);
-                        let value = self.call_driver(command, channel);
-                        if value != Ok(()) {
-                            self.current_app.clear();
-                        }
-                        value
-                    } else {
+            if self.current_app.is_none() {
```

Comment:
We renamed `appid` to `processid`, you might want to update your branch or just change the name of the variable here.